models/blobs/ethereum/transactions.sql

```diff
@@ -0,0 +1,92 @@
+{{ config(
+    schema = 'blobs_ethereum',
+    alias = 'blobs_transactions',    
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['tx_hash', 'blob_versioned_hash'],
+    post_hook='{{ expose_spells(\'["ethereum"]\',
+                                    "project",
+                                    "blobs",
+                                    \'["msilb7","lorenz234"]\') }}'
+)}}
+
+
+SELECT
+    t.block_time
+    , t.block_number
+    , t.hash as tx_hash
+    , t."from" as tx_from
+    , t.to as tx_to
+    , t.gas_used as tx_gas_used
+    , t.gas_price as tx_l1_gas_price
+    , l.base_fee_per_gas AS tx_l1_base_fee
+    , t.blob_versioned_hashes
+    , b.versioned_hash AS blob_versioned_hash
+    , b.used_blob_byte_count -- commented out of blob query for now, since it's slow
+    , b.blob_byte_count AS blobgas_used_by_blob
+    , t.max_fee_per_blob_gas
+    , l.excess_blob_gas
+    , gp.blob_base_fee as blob_base_fee_per_gas
```

Comment:
We should also keep the kzg_commitment field here, so that we can rejoin back to the existing beacon.blobs table.

Likely, all columns in the beacon.blobs table should exist here.